doc/modules/model_evaluation.rst

```diff
@@ -1550,6 +1551,81 @@ Here is a small example of usage of this function:::
     <ftp://ftp.library.noaa.gov/docs.lib/htdocs/rescue/mwr/078/mwr-078-01-0001.pdf>`_,
     Monthly weather review 78.1 (1950)
 
+
+Calibration loss
+----------------
+
+The :func:`calibration_error` function computes the expected and maximum
+calibration losses as defined in [1]_ for binary classes.
+
+Given a set of bins over predicted probabilities, the calibration loss
+measures the overall discrepancy per bin between
+(1) the average predicted probabilities assigned to the positive class, and
+(2) the frequencies of the positive class in the actual outcome in the same
```

Comment:
maybe 'fraction of positive classes in the same bin'? I think 'fraction' is better than frequency is this case..?